wagtail/images/static_src/wagtailimages/scss/add-multiple.scss

```diff
@@ -172,3 +181,28 @@
     }
   }
 }
+
+.confirm-duplicate-upload {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  figure {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 1rem auto;
+
+    img {
+      position: relative;
+    }
+  }
+
+  form {
+    display: inline;
+  }
+
+  + form {
```

Comment:
It's better to rely on classes than HTML structure and elements for these kinds of selectors - it can get quite confusing to work with. This code largely precedes us trying to go for BEM, but try and use that kind of approach - use classes to represent different states, rather than relying on one form being after the other to hide it. I also like to use data-attributes to flag things for js to pick up, rather than css classes.